test/unit/QueueConsumer-test.js

```diff
@@ -0,0 +1,314 @@
+/**
+ * QueueConsumer unit test
+ */
+require('require-self-ref')
+
+const test = require('ava')
```

Comment:
https://github.com/avajs/ava/issues/222 doesn't sound like it's coming anytime soon. Lots of people want it though.